packages/react/src/primitives/Rating/RatingIcon.tsx

```diff
@@ -1,12 +1,12 @@
 import * as React from 'react';
-import { classNames } from '@aws-amplify/ui';
+import { classNames, ComponentClassName } from '@aws-amplify/ui';
 import { Property } from 'csstype';
 
-import { View } from '../View';
+import { BaseComponentProps } from '../types';
 import { StyleToken } from '../types/style';
-import { ComponentClassName } from '@aws-amplify/ui';
+import { View } from '../View';
 
-interface RatingIconProps {
+interface RatingIconProps extends BaseComponentProps {
```

Comment:
Extending `BaseComponentProps` here adds **a lot** of new props to the `Rating` component. Is there a reason we need to add them?